crates/eips/Cargo.toml

```diff
@@ -15,8 +15,12 @@ exclude.workspace = true
 workspace = true
 
 [dependencies]
-alloy-primitives = { workspace = true, features = ["rlp"], default-features = false }
-alloy-rlp = { workspace = true, features = ["derive"], default-features = false }
+alloy-primitives = { workspace = true, features = [
+    "rlp",
+], default-features = false }
```

Comment:
I think we should move default-features=false to all alloy workspace.dependencies